org.eclipse.xtext.xtext.ui.examples/build.xml

```diff
@@ -60,6 +60,7 @@
 		<zip_example subdir="domainmodel"    name="org.eclipse.xtext.example.domainmodel.tests" />
 		<zip_example subdir="domainmodel"    name="org.eclipse.xtext.example.domainmodel.ui" />
 		<zip_example subdir="domainmodel"    name="org.eclipse.xtext.example.domainmodel.ui.tests" />
+		<zip_example subdir="domainmodel"    name="org.eclipse.xtext.example.domainmodel.releng" />
```

Comment:
usual naming convention is .parent

Reply:
it is more than just the parent, it contains also the target platform. I don't like to have too much top-level projects which contain only releng related stuff. Also for repository and feature project.

Reply:
I know it differs from the structure we usually produce with the wizard. But the EMF example wizard can only install separate projects. Better would be to Zip the whole example as one archive and place the parent POM in the root folder.
I had to create a separate project for the releng stuff, thus decided to call it "releng".

Reply:
dont get you. id like to have the name a.b.c.parent as we do it at different places and which is  a common pattern

Reply:
The parent project created with the wizard is physically the root folder of the other projects, which is common Maven style.
We can't do this with the wizard, we have to produce a sibling project that contains the parent pom.xml. Of course, we could name that "parent".

That's the project structure created by the wizard:
![screenshot 40](https://user-images.githubusercontent.com/265597/34262610-2e1dcd86-e66d-11e7-8929-967bdafbccc3.png)

That's the proposed Domainmodel example project structure:
![screenshot 41](https://user-images.githubusercontent.com/265597/34262654-48f9766e-e66d-11e7-8e5d-5da20ffb2dc9.png)

If we would want to use the typical structure, we can't go with the EMF example project anymore. Or we just install the "root" project and have to import the nested projects afterwards.

Reply:
ok go for releng